src/github/pullRequestOverview.ts

```diff
@@ -563,6 +568,89 @@ export class PullRequestOverviewPanel {
 		});
 	}
 
+	private async deleteBranch(message: IRequestMessage<any>) {
+		const branchInfo = await this._pullRequestManager.getBranchNameForPullRequest(this._pullRequest);
+		let actions: (vscode.QuickPickItem & { type: 'upstream' | 'local' | 'remote' })[] = [];
+
+		if (this._pullRequest.isResolved()) {
+			const branchHeadRef = this._pullRequest.head.ref;
+
+			actions.push({
```

Comment:
I'm a bit confused as to why there are two delete remote branch options, this one and the one below. Shouldn't there only ever be one that always respects the preferredRemoteDeletionMethod setting?

Reply:
The second is deleting `remote` while this one is deleting `remote branch` which is the branch stored in upstream.